Tasks/AzureResourceGroupDeploymentV2/task.json

```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 141,
-        "Patch": 5
+        "Minor": 142,
```

Comment:
Can you mark the minor version to 145? In this task, we are aligning the minor version to our sprint numbers.

Reply:
Done.